src/drivers/esp8266_drv.c

```diff
@@ -400,7 +405,7 @@ static void socket_disconnect_handler(const size_t chan_id)
 	/* Check if the close op is in progress.  If so, take no action */
 	struct channel_sync_op *cso = &esp8266_state.comm.close_op;
 	if (cso->chan_id == chan_id)
-		return;
+	        return;
```

Comment:
Is your editor using spaces or tabs?  I forget which one we use but best to check that its consistent less we mess up the whole damn structure

Reply:
spaces. I think there's a mix of indentation originally, could use a bulk reformat.